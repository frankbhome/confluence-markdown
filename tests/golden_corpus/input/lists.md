# Lists Example

## Unordered List

- First item
- Second item
- Third item

## Ordered List

1. First numbered item
2. Second numbered item
3. Third numbered item

## Mixed Content

- Item with **bold** text
- Item with `inline code`
- Item with [a link](https://example.com)
