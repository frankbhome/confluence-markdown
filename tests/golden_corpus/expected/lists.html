<h1>Lists Example</h1>

<h2>Unordered List</h2>

<ul><li>First item</li>
<li>Second item</li>
<li>Third item</li>
</ul>

<h2>Ordered List</h2>

<ol><li>First numbered item</li>
<li>Second numbered item</li>
<li>Third numbered item</li>
</ol>

<h2>Mixed Content</h2>

<ul><li>Item with <strong>bold</strong> text</li>
<li>Item with <code>inline code</code></li>
<li>Item with <a href="https://example.com">a link</a></li>
</ul>
